packages/vite/src/plugins/plugin.ts

```diff
@@ -66,9 +66,10 @@ export const createNodes: CreateNodes<VitePluginOptions> = [
 
     options = normalizeOptions(options);
 
-    const hash = calculateHashForCreateNodes(projectRoot, options, context, [
-      getLockFileName(detectPackageManager(context.workspaceRoot)),
-    ]);
+    const hash =
+      calculateHashForCreateNodes(projectRoot, options, context, [
+        getLockFileName(detectPackageManager(context.workspaceRoot)),
+      ]) + configFilePath;
```

Comment:
Rather than appending the config file, can we include them in the `additionalGlobs` array or `options` object to be hashed?

Reply:
I asked @AgentEnder about this and he felt it was better to append the configFilePath

Reply:
Will add a comment about this